### DISCUSSION TITLE
Unable to reuse events ?

### DISCUSSION THREAD

**Discussion by jsantosi** [liedekef/events-made-easy#162] [Q&A]
I have past events that I intended to reuse, because it's the same description, location forms, and I also use their ids in the code of my functions to limit registrations. But I saw that if I change the date of the event it automatically changes the date for those who have participated in another time, ie, the user participated in the event last month and there in the "My account" page he has access to his schedules through the shortcode [eme_mybookings], then I as administrator decide that the same event will happen this month and along with it I release pages with gifts and so on, each event has something new. This same user that participated last month sees the new date in his own schedule and can have access to things that he would not have, and in my mind this should not happen, because the user last month participated in this event, there was the record in the database and that is the record of the user, any change I make in the future event should not impact this data that is already recorded. Is there any way to solve this?

**Comment by liedekef** [comment]
Changing the date of an existing event does what it does: it just changes the date, it does not void any registrations.
Either you remove those registrations, or click on the "copy" icon in the list of events.

**Comment by jsantosi** [comment]
It does not unsubscribe but the user who has already participated will see the events that he has already signed up for and will have one there with a future date to happen but he did not sign up again, i.e., the user would be confused at least because how can a past event is on my dashboard as a future event, if [eme_mybookings] is already saved in the database for that user with his options, this should not happen. I made a text with custom fields because I have events that I put times there and when I change them it doesn't show up changed for the user because he already made his choice in the past. In my mind the same should happen for date changes for events, so I wouldn't have dozens of events with practically the same information and only with different dates.

**Reply by liedekef** [reply]
While there's a logic to every behavior, my answer stays the same. Changing a past event to a future event and removing all bookings would also remove all info on payments, number of participants, confuse google etc ...
That's then your choice to make, not mine. I recommend to copy the event, it's only 1 click and does what you want.
